index.html

```diff
@@ -1,276 +1,240 @@
-    </header>
-   <!--main section--> 
-    <div id="main_section" class="grid" >
-     <div class="container">
-      <div class="h1">I’m Patrick Ndimukulaga Glad to see you!</div>
-      <p class="p">
-        I’m a software developer! I can help you build a product , feature or
-        website Look through some of my work and experience! If you like what
-        you see and have a project you need coded, don’t hestiate to contact
-        me.
-      </p>
-
-      <div>
-        <a href="#" class="media_p">LET'S CONNECT</a>
-        <ul class="media_ul">
-          <li class="media_li">
-            <a href="#"><img src="./images/Twitter-icon.svg" alt="Twitter icon"/></a>
-          </li>
-          <li class="media_li">
-            <a href="#"><img src="./images/Linkedin-icon.svg" alt="Linkedin icon"/></a>
-          </li>
-          <li class="media_li">
-            <a href="#"><img src="./images/Medium-icon.svg" alt="Medium icon"/></a>
-          </li>
-          <li class="media_li">
-            <a href="#"><img src="./images/Github-icon.svg" alt="Github icon"/></a>
-          </li>
-          <li class="media_li"><a href="#"><img src="./images/Vector.svg" alt="Vector icon"/></a>
-          </li>
-        </ul>
-      </div>
-      </div>
-    </div>
-    
-    <section class="simi">
-      <div class="title grid">
-       
-        <div>
-          <img src="./images/Snapshoot_Portfolio.png" alt="portfolio"/>
-        </div>
-        <div class="title-text">
-          <div class="t-text">
-            <div class="port">
-              <h4>Tonic</h4>
-
-              <ul class="ul">
-                <li class="first-child">CANOPY</li>
-                <li>Back End Dev</li>
-                <li>2015</li>
-              </ul>
+  <body>
+    <div id="bk_container">
+        <header class="navbar">
+            <img src="./images/Portfolio_Logo.png" alt="My Logo" class="logo"/>
+            <nav class="nav">
+                <ul>
+                    <li><a href="#" class="menu_li" >Portfolio</a></li>
+                    <li><a href="#" class="menu_li">About</a></li>
+                    <li><a href="#" class="menu_li">Contact</a></li>
+                </ul> 
+            </nav>
+            <img src="./images/Enabled.png" alt="menu" class="menu-icon"/>
+        </header>
+        <div class="port_container">
+            <div class="main_section">
+                <div class="section_header">I’m Patrick Ndimukulaga<br>Glad to see you!</div>
+                <p class="section_text">
+                  I’m a software developer! I can help you build a product , feature or
+                  website Look through some of my work and experience! If you like what
+                  you see and have a project you need coded, don’t hestiate to contact
+                  me.
+                </p>
+                <div class="media">
+                    <a href="#" class="media_p">LET'S CONNECT</a>
+                    <ul class="media_ul">
```

Comment:
MOBILE VERSION:

- [ ] Kindly reduce the space for the entire mobile version from the right side.
Figma design:
<img width="304" alt="Screenshot 2022-05-05 at 1 40 19 AM" src="https://user-images.githubusercontent.com/70269507/166842142-3e3cdb10-e3d4-43d0-95bd-06b89ee260a7.png">

Your design:
<img width="416" alt="Screenshot 2022-05-05 at 2 35 01 AM" src="https://user-images.githubusercontent.com/70269507/166842157-424c547c-d43c-433e-ad81-482186665260.png">